style.css

```diff
@@ -34,11 +35,94 @@ html {
   font-family: var(--font-secondary);
   font-weight: 300;
   font-size: 24px;
-  color: var(--gray-dark);
 }
 
- /*@media (max-width: 1024px) {
-}*/
+.hero {
+  margin-top: 20px;
+}
+.hero__content {
+  display: flex;
+  gap: 60px;
+}
+.hero__label {
+  font-family: var(--font-secondary);
+  font-size: 34px;
+  font-weight: 700;
+  display: flex;
+  align-items: flex-end;
+}
+.hero__label::before {
+  content: "";
+  height: 0.5em;
+  width: 48px;
+  border-top: 2px solid var(--gray-dark);
+  margin-right: 12px;
+}
+.hero__title {
+  margin-top: 20px;
+  font-size: 74px;
+  font-weight: 400;
+}
+.hero__title--name {
+  font-weight: bold;
+}
+.hero__subtitle {
+  margin-top: 20px;
+  font-size: 40px;
+  font-weight: 400;
+}
+.hero__text {
+  margin-top: 20px;
+  font-size: 24px;
+}
+
+.hero__scroll {
+  margin-top: 56px;
+  display: flex;
+  align-items: center;
+}
+.social__items {
+  display: flex;
+  gap: 32px;
+}
+.scroll {
+  display: flex;
+  margin: auto;
+  flex-direction: column;
+  cursor: pointer;
+}
+.scroll__text {
+  font-size: 18px;
+  font-weight: 300;
+}
+.scroll__icon {
+  margin-top: 12px;
+  height: 32px;
+}
+.space {
+  width: 100px;
+  height: 65px;
+}
+
+@media (max-width: 1024px) {
+  .hero__content {
+    display: block;
+  }
+  .avatar {
+    margin-top: 56px;
```

Comment:
Maybe it would be better to put the margin-top on .hero__image instead of here, because look at what happens now:
![image](https://user-images.githubusercontent.com/68795596/202469477-292832bb-0f1d-4f66-b991-b1ad58f9293b.png)

Reply:
And actually you could even remove the div and add the class to the picture element, because you only have one element inside the div.